features/events/rsvping_event.feature

```diff
@@ -0,0 +1,55 @@
+@javascript
+Feature: RSVPing AV events
+  As an event creator
+  In order to see people planning to attend my event
+  I would like to be able to confirm attendence
+
+  Background:
+    Given the following users exist
+      | first_name | last_name | id  | password  | email                  |
+      | Alice      | Jones     | 401 | 12345678  | alicejones@hotmail.com |
+      | John       | Doe       |     | password  | john@doe.com           |
+    And following events exist:
+      | name       | description             | category        | creator_id | start_datetime          | duration | repeats | time_zone | project | repeats_weekly_each_days_of_the_week_mask | repeats_every_n_weeks | creator_attendance |
+      | ClientMtg  | Weekly client meeting   | ClientMeeting   | 401        | 2018/07/19 11:00:00 UTC | 150      | never   | UTC       |         |                                           |                       | true       |
+      | Meeting    | Weekly meeting          | Meeting         | 401        | 2018/07/21 11:00:00 UTC | 150      | weekly  | UTC       |         | 31                                        | 1                     | false      |
+
+  Scenario: Non-logged in users should not see Cannot Attend button
```

Comment:
we might be able to dry these out with Scenario Outlines?  but no rush - could be a refactoring ticket so we can get this deployed ...